specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/examples/watchlists/DeleteWatchlist.json

```diff
@@ -4,10 +4,14 @@
     "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
     "resourceGroupName": "myRg",
     "workspaceName": "myWorkspace",
-    "watchlistAlias": "highValueAsset"
+    "operationalInsightsResourceProvider": "Microsoft.OperationalInsights",
+    "alias": "highValueAsset"
   },
   "responses": {
     "200": {},
+    "202": {
+      "body": {}
```

Comment:
I think you need to add headers with Azure-AsycnOperation here.  the current Lint tool has a bug, so it's not catching it. I was asked to add to our swagger examples in our own PR.

Reply:
Sounds good. Added it. Please let me know if you're able to approve with that change as we're coming up on the change deadline of Feb 28th. Thanks!